pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -118,7 +113,13 @@ func haveOverlap(a1, a2 []string) bool {
 // - Ceph RBD forbids if any two pods share at least same monitor, and match pool and image, and the image is read-only
 // - ISCSI forbids if any two pods share at least same IQN and ISCSI volume is read-only
 func (pl *VolumeRestrictions) Filter(ctx context.Context, _ *framework.CycleState, pod *v1.Pod, nodeInfo *framework.NodeInfo) *framework.Status {
-	for _, v := range pod.Spec.Volumes {
+	for i := range pod.Spec.Volumes {
```

Comment:
It's no harm to keep the original `for _, v := range pod.Spec.Volumes` logic, no?

Reply:
why should we copy the object twice?

Reply:
True, but I'd argue the cost (the size of `Volume` is trivial) is trivial.

/lgtm
anyway